CHANGELOG.md

```diff
@@ -9,10 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `mypy` for search space and objectives
 - Class hierarchy for objectives
 - Deserialization is now also possible from optional class name abbreviations
-- Hypothesis strategies for acquisition functions
 - `Kernel` base class allowing to specify kernels
 - `MaternKernel` class can be chosen for GP surrogates
-- `hypothesis` strategies and roundtrip test for kernels and objectives
+- `hypothesis` strategies and roundtrip test for kernels, objectives and acquisition
+  functions
+- New acquisition functions: `qSR`, `qNEI`, `LogEI`, `qLogEI`, `qLogNEI`
```

Comment:
@Scienfitz just to avoid confusion: Is this already ready for review or does it make more sense until the other aspects/issues are implemented/solved?

Reply:
not much has changed since your question but i guess now its ready